### PR TITLE
Freebsd pkg addon

### PR DESCRIPTION
**Pull request by DamianSzymanski**

### CODE REVIEW

File: lib/travis/yml/schema/def/addon/pkg.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    module Schema
+      module Def
+        module Addon
+          class Pkg < Addon
+            register :pkg
+
+            def define
+              prefix :packages
+
+              map :packages, to: :seq, alias: :package, summary: 'Package names', eg: 'cmake'
```
Comment (svenfuchs) [inline lib/travis/yml/schema/def/addon/pkg.rb]: i'd suggest using `names` here `pkg` already suggests "package", so "package names" would read better than "package package". 

also, i'd suggest not adding aliases unless we have to.

Comment (DamianSzymanski) [reply lib/travis/yml/schema/def/addon/pkg.rb]: It was suggestion from freebsd community c (like apt)
```addons:
  pkg:
    packages:
    - cmake
    - vim
```
and alias
```addons:
  pkg:
    package: cmake
```

Comment (michal-at-travisci) [reply lib/travis/yml/schema/def/addon/pkg.rb]: On Ubuntu builds we keep aliases as follows:
```
addons:
  apt:
    packages:
```

- so FreeBSD community proposed something consistent. The `apt` is replaced with `pkg` in FreeBSD, which may be not so fortunate, but I'm for keeping it consistent with linux based mechanism - less confusion for users

Comment (michal-at-travisci) [reply lib/travis/yml/schema/def/addon/pkg.rb]: and we already made it consistent on macos:

```
addons:
  homebrew:
    packages:
```

so if we introduce something different now, we should allow it for all other OSes - I'd prefer to give less options to the users

**Pull request merged**